sdk/executor/src/main/java/com/mesosphere/sdk/executor/ProcessTask.java

```diff
@@ -39,18 +37,8 @@ public static ProcessTask create(
             ExecutorDriver executorDriver,
             Protos.TaskInfo taskInfo,
             boolean exitOnTermination) {
-        return create(executorDriver, taskInfo, getProcess(taskInfo), exitOnTermination);
-    }
-
-    public static ProcessBuilder getProcess(TaskInfo taskInfo) {
-        CommandInfo commandInfo = taskInfo.getCommand();
-        String cmd = commandInfo.getValue();
-
-        ProcessBuilder builder = new ProcessBuilder("/bin/sh", "-c", cmd);
```

Comment:
I think it'd make more sense to specifically go with `/bin/bash` as done here if only because of that issue -- to make the expected environment more consistent.